deployments/fargate/README.md

```diff
@@ -0,0 +1,95 @@
+# AWS Fargate Deployment
+Familiarity with AWS Fargate (Fargate) is assumed. Consult the 
+[User Guide for AWS Fargate](https://docs.aws.amazon.com/AmazonECS/latest/userguide/what-is-fargate.html) for further reading.
+
+The
+[Splunk OpenTelemetry Collector](https://github.com/signalfx/splunk-otel-collector)
+(splunk-collector or collector) can be deployed to an ECS Task as a standalone container 
+or as sidecar container alongside the application containers that you want to monitor.
+
+Find the Docker image repository for the collector
+[here](https://quay.io/repository/signalfx/splunk-otel-collector?tab=tags).
+The latest tag is `quay.io/signalfx/splunk-otel-collector:latest`.
+
+## Standalone Container
+As a standalone container, you add the container definition for the collector to a
+task that is separate from the task containing the application containers that you 
+want to monitor. You configure the collector to use extension
+[Amazon Elastic Container Service Observer (ecsobserver)](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/extension/observer/ecsobserver#amazon-elastic-container-service-observer).
+to discover targets in running tasks, filtered by service names, task definitions
+and container labels. The ecsobserver is currently limited to discovering
+prometheus targets. Also, the task role must include the read-only permissions below. 
+The permissions can be added to a customer-managed policy that is attached to the
+task role.
+
```

Comment:
Suggested change: "You can configure the collector to use the use the Amazon Elastic Container Service Observer (ecsobserver) extension to discover targets in running tasks, filtered by service names, task definitions, and container labels. The ecsobserver is currently limited to discovering Prometheus targets. The task role must also include the read-only permissions shown below. The permissions can be added to a customer-managed policy that is attached to the task role."

Reply:
I rewrote the entire readme